catalog/region-specific-policies.yaml

```diff
@@ -0,0 +1,23 @@
+- sid: "DenyRegionUsage"
+  effect: "Deny"
+  not_actions:
+    - "cloudfront:*"
+    - "iam:*"
+    - "route53:*"
+    - "support:*"
+    - "organizations:*"
+    - "waf:*"
+    - "budgets:*"
+    - "globalaccelerator:*"
+    - "cur:*"
+    - "ce:*"
+    - "directconnect:*"
+  condition:
+    - test: "StringNotEqualsIgnoreCase"
+      variable: "aws:RequestedRegion"
+      values:
+        # The regions that can't be used for deploying infrastructure
+        - "${region1_lockdown}"
```

Comment:
Why only two regions?
Can we use more here?

Reply:
yes, that was just as an example

Reply:
can we make it a list and provide it in `${regions_lockdown}` ?

Reply:
if we hardcode just two regions, then it will be possible to use only 2.
Why not 3? Or we can add 50 just in caae.
List would be better here (not sure if it will work with the params in YAML)

Reply:
open to feedback, we could either delete this rule or add a better explanation of why is only two.
The reason there is only two ways to illustrate how that could be achieved with parameters but not necessarily a comprehensive list of the region that one could block. This type of setting is very specific to a different types of industries and government regulations.

Reply:
Hi @jamengual 
> # The regions that can't be used for deploying infrastructure
CMIIW isn't it the inverse ? "The regions that can be used"

Reply:
as with DenyAllOutsideEU  @  https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps_examples.html#examples_general

Reply:
rule deleted for now